app/migrations.go

```diff
@@ -31,7 +30,6 @@ func (s *Server) doAdvancedPermissionsMigration() {
 
 	mlog.Info("Migrating roles to database.")
 	roles := model.MakeDefaultRoles()
-	roles = utils.SetRolePermissionsFromConfig(roles, s.Config(), s.License() != nil)
```

Comment:
Slightly unrelated but wondering if we should be keeping 3+ years old migration code around. I am thinking this could be an opportunity to just drop it (in a separate PR of course). 

@mkraft Thoughts?

Reply:
> Slightly unrelated but wondering if we should be keeping 3+ years old migration code around. I am thinking this could be an opportunity to just drop it (in a separate PR of course).
> 
> @mkraft Thoughts?

Generally sounds like a good idea. Do you want to create a ticket @streamer45 and part of the ticket can be to investigate the details of whether it's feasible?

Reply:
Sure :+1: [(MM-37842)](https://mattermost.atlassian.net/browse/MM-37842)